packages/dialog/src/index.js

```diff
@@ -83,12 +86,18 @@ const DialogInner = React.forwardRef(function DialogPortal(
   );
 });
 
+if (__DEV__) {
+  DialogOverlay.displayName = "DialogOverlay";
+}
+
+////////////////////////////////////////////////////////////////////////////////
 export const DialogContent = React.forwardRef(function DialogContent(
   { onClick, onKeyDown, ...props },
   forwardedRef
 ) {
   return (
     <div
+      role="dialog"
```

Comment:
Seems weird this was missing before... isn't it kind of important?

Reply:
I thought so too, but following the spec it should be there https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html